CloudFormation/arkcase.yml

```diff
@@ -1452,16 +1543,530 @@ Resources:
+            Principal:
+              Service: ecs-tasks.amazonaws.com
+            Action: sts:AssumeRole
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy
+      Policies:
+        - PolicyName: AlfrescoTaskExecutionPolicy
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Action: ssm:GetParameters
+                Resource:
+                  - !GetAtt Pki.Outputs.AlfrescoKeyParameterArn
+                  - !GetAtt Pki.Outputs.AlfrescoCertParameterArn
+                  - !GetAtt Pki.Outputs.IntermediateCaCertParameterArn
+                Effect: Allow
+      Tags:
+        - Key: Name
+          Value: !Sub alfresco-task-execution-role-${Project}-${Env}
+        - Key: Env
+          Value: !Ref Env
+        - Key: Project
+          Value: !Ref Project
+        - Key: Service
+          Value: alfresco
+        - Key: Component
+          Value: iam
+        - Key: ManagedBy
+          Value: CloudFormation
+
+  AlfrescoTaskRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: ecs-tasks.amazonaws.com
+            Action: sts:AssumeRole
+      Policies:
+        - PolicyName: AlfrescoTaskPolicy
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Action: secretsmanager:GetSecretValue
+                Resource:
+                  - !GetAtt AlfrescoDatabaseUser.Outputs.UserSecretArn
+                  - !GetAtt Amazonmq.Outputs.AlfrescoUserSecretArn
+                Effect: Allow
+
+              - Action: route53:ChangeResourceRecordSets
+                Resource: !Sub arn:aws:route53:::hostedzone/${PrivateHostedZone}
+                Effect: Allow
+      Tags:
+        - Key: Name
+          Value: !Sub alfresco-task-role-${Project}-${Env}
+        - Key: Env
+          Value: !Ref Env
+        - Key: Project
+          Value: !Ref Project
+        - Key: Service
+          Value: alfresco
+        - Key: Component
+          Value: iam
+        - Key: ManagedBy
+          Value: CloudFormation
+
+  AlfrescoTaskDefinition:
+    # NB: Containers in an `awsvpc` task can access each other on `localhost`
+    Type: Custom::TaskDefinition
+    Properties:
+      ServiceToken: !GetAtt TaskDefinitionResourceLambda.Arn
+      family: alfresco
+      cpu: 2048
+      memory: 8192
+      executionRoleArn: !GetAtt AlfrescoTaskExecutionRole.Arn
+      taskRoleArn: !GetAtt AlfrescoTaskRole.Arn
+      requiresCompatibilities: [ FARGATE ]
+      networkMode: awsvpc
+      volumes:
+        - name: alfresco-efs
+          efsVolumeConfiguration:
+            fileSystemId: !Ref AlfrescoEfs
+      containerDefinitions:
+        - name: acs-repository
+          image: !Sub ${AWS::AccountId}.dkr.ecr.${AWS::Region}.amazonaws.com/acs-repository:dev4
```

Comment:
Where is the code to create and push this image?  If the lambda / image source code is open source on GitHub, we could add a comment here with a link to the source.